lib/response.js

```diff
@@ -64,7 +64,18 @@ var charsetRegExp = /;\s*charset\s*=/;
  */
 
 res.status = function status(code) {
-  this.statusCode = code;
+  if (code > 99 && code < 1000) {
+    if (typeof code === 'string') {
+      console.log('Response status code set to a value with type String. ' +
+        'Express v5+ will throw on status codes which are not integers of type Number.')
```

Comment:
The deprecation message should really be worded like all the rest, namely:

1. Not refer to a future version of this module (i.e. do not say "Express vFuture will ....")
2. Inform the user what they are supposed to do instead; in other words what is their course of actions (i.e. "string "200" is deprecation, pass as a number 200").

If it were me, as an example, I would word this specific message as `deprecate("res.status(" + JSON.stringify(code) + "): use res.status(" + code + ") instead")`. This would, given the call `res.status("200")` print the message `express deprecated res.status("200"): use res.status(200) instead app.js:4:54`

Reply:
Updated the dep message